tutoraspects/templates/aspects/build/aspects/entrypoint

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+
+if [ -z "${DBT_SSH_KEY+x}"]
+then
+  mkdir -p /root/.ssh
+  echo "${DBT_SSH_KEY}" | tr -d '\r' > /root/.ssh/id_rsa
+  chmod 600 /root/.ssh/id_rsa
+  eval `ssh-agent -s`
+
+  ssh -o StrictHostKeyChecking=no git@github.com || true
+  ssh-add /root/.ssh/id_rsa
+fi
+
+rm -rf aspects-dbt
+
+echo "Installing aspects-dbt"
+echo "git clone -b ${DBT_BRANCH} ${DBT_REPOSITORY}"
+git clone -b ${DBT_BRANCH} ${DBT_REPOSITORY} aspects-dbt
+
+cd aspects-dbt
+
```

Comment:
To get the exposures you'll need to add the dbt-state volume to this service like we do in local-docker-compose-jobs-service and then just copy the generated file in (but don't fail if it doesn't exist) with something like:

```
echo "Trying to copy the Superset exposures file generated by the collect-dbt-lineage command"
cp /app/aspects/dbt_state/superset_exposures.yaml ./models || :
```

Reply:
I'm storing those files on ClickHouse but that's WIP. It's a small refactor but requires some time to finish.